testsuite/run_forge.sh

```diff
@@ -38,7 +38,7 @@ FORGE_RUNNER_MODE=${FORGE_RUNNER_MODE:-k8s}
 FORGE_NAMESPACE_KEEP=${FORGE_NAMESPACE_KEEP:-false}
 FORGE_NAMESPACE_REUSE=${FORGE_NAMESPACE_REUSE:-false}
 FORGE_ENABLE_HAPROXY=${FORGE_ENABLE_HAPROXY:-false}
-FORGE_TEST_SUITE=${FORGE_TEST_SUITE:-land_blocking}
+FORGE_TEST_SUITE=${FORGE_TEST_SUITE:-bench_with_fullnode}
```

Comment:
@bchocho - Just learned that you need to change this to 

```
FORGE_TEST_SUITE=bench_with_fullnode

```
This is due to recent change where GH started passing this default value.